modules/validations/ambiguous_crossing_tags.js

```diff
@@ -1,52 +1,67 @@
 
 import { actionChangeTags } from '../actions/change_tags';
-import { osmRoutableHighwayTagValues } from '../osm/tags';
 import { ValidationIssue, ValidationFix } from '../core/lib';
-
+import { utilHashcode } from '@rapid-sdk/util';
```

Comment:
@bhousel - I have questions on what significance the hashCodes have for each validation. There may be some subtle bugs here if I calc'd the wrong thing into a hashCode.

Reply:
Oh - `utilHashCode` is just for computing a very fast unique number from any string input. Our SDK might be using one of the implementations from [here](https://gist.github.com/hyamamoto/fd435505d29ebfa3d9716fd2be8d42f0)

It's just for making a unique ID out of something.  

![Screenshot 2023-11-08 at 9 43 15 AM](https://github.com/facebook/Rapid/assets/38784/2446bd47-caf5-4ecc-88fb-84cdd8c26346)

So a validator that might raise multiple issues about the same entity, the hash gets included in the issueID so code can tell them apart:

https://github.com/facebook/Rapid/blob/1185147b6b6ef6f8435da8c8a3d55417bc4fb09c/modules/core/lib/ValidationIssue.js#L67-L74

(Now that we are targeting modern browsers, we should maybe use WebCrypto API to compute a digest - this might be faster?)